inc/Frame.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+class Frame
+{
+    int value_;
+    bool isStrike;
```

Comment:
Angan7a, I think this bool variables can be const, cause information strike and spare will be known when object will be created